### ISSUE TITLE
No result displayed - "Attempt to update previously set global instance"

### ISSUE THREAD

**Issue by Otiel** [CoreyHayward/PowerToys-Run-ClipboardManager#7]
Plugin does not work at all. Nothing is displayed when I type "c:".

I'm getting the following exception in the log :

```
[2024-10-09 10:54:01.5778] [ERROR] [D:\a\_work\1\s\src\modules\launcher\PowerLauncher\Plugin\PluginManager.cs::240]
-------------------------- Begin exception --------------------------
Message: Exception for plugin <ClipboardManager> when query <c:>

Exception full name  : System.InvalidOperationException
Exception message    : Attempt to update previously set global instance.
Exception stack trace:
   at System.Runtime.InteropServices.ComWrappers.RegisterForTrackerSupport(ComWrappers instance)
   at WinRT.ComWrappersSupport.get_ComWrappers()
   at WinRT.ComWrappersSupport.CreateRcwForComObject[T](IntPtr ptr, Boolean tryUseCache)
   at WinRT.ComWrappersSupport.CreateRcwForComObject[T](IntPtr ptr)
   at WinRT.MarshalInspectable`1.FromAbi(IntPtr ptr)
   at WinRT.MarshalInterface`1.FromAbi(IntPtr ptr)
   at ABI.Windows.ApplicationModel.DataTransfer.IClipboardStatics2Methods.GetHistoryItemsAsync(IObjectReference _obj)
   at Windows.ApplicationModel.DataTransfer.Clipboard.GetHistoryItemsAsync()
   at Community.PowerToys.Run.Plugin.ClipboardManager.Main.<>c.<<GetTextItemsFromClipboardHistory>b__14_0>d.MoveNext()
--- End of stack trace from previous location ---
   at Community.PowerToys.Run.Plugin.ClipboardManager.Main.RunSync[T](Func`1 func)
   at Community.PowerToys.Run.Plugin.ClipboardManager.Main.GetTextItemsFromClipboardHistory()
   at Community.PowerToys.Run.Plugin.ClipboardManager.Main.Query(Query query)
   at PowerLauncher.Plugin.PluginManager.<>c__DisplayClass23_0.<QueryForPlugin>b__0()
   at Wox.Infrastructure.Stopwatch.Debug(String message, Action action)
   at PowerLauncher.Plugin.PluginManager.QueryForPlugin(PluginPair pair, Query query, Boolean delayedExecution)
Exception source     : System.Private.CoreLib
Exception target site: Void RegisterForTrackerSupport(System.Runtime.InteropServices.ComWrappers)
Exception HResult    : -2146233079
-------------------------- End exception --------------------------
```

Using 
* PowerToys 0.85.1
* Windows 10 22H2

**Comment by mswift98** [comment]
I have the same issue. Nothing happens after typing the "c:".

PowerToys 0.85.1
ClipboardManager Plugin v0.5.0
Windows 11 23H2

**Comment by CoreyHayward** [comment]
@Otiel @mswift98 Thanks for reporting this issue, looks like this issue has been introduced with PowerToys 0.85.0. 
I can recreate it and downgrading to 0.84.1 the plugin seems to work correctly. 
I'll see what I can do in terms of getting this fixed ASAP but for now I can only suggest downgrading PowerToys if you use the plugin regularly (sorry I know its not the best solution, but works for now at least).

**Comment by Otiel** [comment]
No problem, I'll patiently wait for a fix, take the time you need. ❤

Thanks for the quick feedback.

**Comment by CoreyHayward** [comment]
This issue has now been fixed in the latest release [v0.6.0](https://github.com/CoreyHayward/PowerToys-Run-ClipboardManager/releases/tag/v0.6.0)

Thanks for your patience!

**Comment by Otiel** [comment]
Thanks for the quick fix, I confirm it works fine now. ❤️
